flocker_bb/steps.py

```diff
@@ -22,6 +23,27 @@
 
 TWISTED_GIT = b'https://github.com/twisted/twisted'
 
+flockerBranch = Interpolate("%(src:flocker:branch)s")
+
+buildNumber = Interpolate("%(prop:buildNumber)s")
```

Comment:
It is. `Interpolate` is a type representing something to be rendered at build time.